app/templates/components/gh-members-lab-setting.hbs

```diff
@@ -64,18 +64,38 @@
         </div>
 
         {{#liquid-if this.membersPricingOpen}}
+            <div class="w-100 w-50-l flex flex-column flex-row-ns mt8">
+                <div class="w-100 w-50-ns mr3-ns">
+                    {{#gh-form-group class="for-select"}}
+                        <label class="fw6 f8"for="currency">Plan currency</label>
+                        <p>{{this.selectedCurrency.value}}</p>
+                        <span class="gh-select">
+                            {{one-way-select this.selectedCurrency
+                                id="currency"
+                                name="currency"
+                                options=(readonly this.currencies)
+                                optionValuePath="value"
+                                optionLabelPath="label"
+                                update=(action "setSubscriptionSettings" "currency")
```

Comment:
This is where it was falling down.

- `options` is an array of currency objects, which it's necessary to specify the path for the value and label
- `this.selectedCurrency` needs to match one of the objects in the `options` 
- `update` passes the full currency object through to the action but in the action we were [only grabbing the value](https://github.com/TryGhost/Ghost-Admin/pull/1430/commits/74323c9431aa198e11391eddff195c70541894b6#diff-216299560d5da7f490965c492f9b0186L143) and setting that to `this.selectedCurrency` which then no longer matches one of the objects in `options`

Reply:
Makes sense. Thanks for explaining :)